src/renderer/components/instance/SavesPage.ts

```diff
@@ -0,0 +1,13 @@
+import { customElement, html, LitElement, property, TemplateResult } from "lit-element";
+import type Instance from "../../Instance";
+
+@customElement("saves-page")
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=6041480)